sdk/search/azure-search-documents/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,59 @@
+# Troubleshooting Azure Cognitive Search SDK Issues
+
+The `azure-search-documents` package provides APIs for operations on the 
+[Azure Cognitive Search](https://docs.microsoft.com/azure/search/search-what-is-azure-search) cloud service.
+
+## Table of Contents
+
+* [Identifying and Troubleshooting Issues by Response Code](#troubleshooting-issues-by-response-code)
+    * [HTTP 207 Errors](#207-multi-status)
+    * [HTTP 404 Errors](#404-not-found)
+    * [HTTP 429 Errors](#429-too-many-requests)
+* [Unexpected search query results](#unexpected-search-query-results)
+
+## Troubleshooting Issues By Response Code
+
+See [this page](https://docs.microsoft.com/rest/api/searchservice/http-status-codes) for the common response status 
+codes sent by the Azure Cognitive Search service.
+
+### 207 Multi-Status
+
+This response status indicates a partial success for an indexing operation. Some documents were successfully processed, but at least one failed. Details of the failed documents are present in the individual `IndexingResult` objects within the `IndexDocumentsResult`. 
+
+### 403 Forbidden
+
+Returned when you pass an invalid api-key. Search service uses two types of keys to control access: admin (read-write) 
+and query (read-only). The **admin key** grants full rights to all operations, including the ability to manage the service, 
+create and delete indexes, indexers, and data sources. The **query key** grants read-only access to indexes and documents. 
+Ensure that the key used for an API call provides sufficient privileges for the operation. See [here](https://docs.microsoft.com/azure/search/search-security-api-keys) 
+for details about managing API keys.
+
+If you are using the `azure-identity` package to authenticate requests to Azure Cognitive Search, please see our [troubleshooting guide](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/TROUBLESHOOTING.md).
+
+### 404 Not Found
+
+Returned when a resource does not exist on the server. If you are managing or querying an index, check the syntax and 
+verify the index name is specified correctly.
+
+### 429 too many requests
+
+If this error occurs while you are trying to create an index, it means you already have the maximum number of indexes 
+allowed for your pricing tier. A count of the indexes stored in Azure Cognitive Search is visible in the search service 
+dashboard on the [Azure portal](https://portal.azure.com/). To view the indexes by name, click the Index tile. 
+Alternatively, you can also get a list of the indexes by name using the [listIndexNames() method](https://docs.microsoft.com/java/api/com.azure.search.documents.indexes.searchindexclient.listindexnames?view=azure-java-stable).
+
+If this error occurs during document upload, it indicates that you've exceeded your quota on the number of documents per 
```

Comment:
Is there a link to the quota limits that can be added here?

Reply:
Added